workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```diff
@@ -46,10 +46,66 @@ amenities:
+    or (healthcare = blood_donation)
+    or (healthcare:speciality = vacciniation)
+    or (highway = raceway)
+    or (historic in (aircraft, aqueduct, archaeological_site, battlefield, boundary_stone, building,
+    castle, cannon, city_gate, citywalls, farm, fort, gallows, highwater_mark, locomotive, manor,
+    memorial, milestone, monastery, monument, optical_telegraph, pillory, ruins, rune_stone, ship,
+    tomb, wayside_cross, wayside_shrine, wreck))
+    or (leisure in (adult_gaming_centre, amusement_arcade, beach_resort, bandstand, bird_hide, common,
+    dance, dog_park, firepit, fishing, fitness_centre, garden, golf_course, hackerspace,
+    horse_riding, ice_rink, marina, miniature_golf, nature_reserve, park, picnic_table, pitch, playground,
+    sauna, slipway, sports_centre, stadium, summer_camp, swimming_area, swimming_pool, track, turkish_bad,
+    water_park, wildlife_hide))
+    or (natural in (beach, cave_entrance, geyser, peak, rock, saddle, spring, volcano,
+    water))
+    or (public_transport in (platform, stop_position, station, stop_area))
+    or (railway in (halt, station, tram_station))
+    or (shop in (agrarian, alcohol, antiques, art, bag, bakery, beauty, bed, beverages,
+    bicycle, books, boutique, brewing_supplies, business_machines, butcher, cafe,camera,
+    candles, car, car_parts, carpet, curtain, cheese, chemist, chocolate,
+    clothes, coffee, computer, confectionery, convenience, copyshop, cosmetics,
+    dairy, deli, department_store, doityourself, dry_cleaning, electrical, electronics,
+    erotic, estate_agent, e-cigarette, farm, fashion, fishing, florist, funeral_directors,
+    furniture, games, garden_centre, garden_furniture, gas, general, gift, glaziery, greengrocer, grocery,
+    hairdresser, hairdresser_supply, hardware, hearing_aids, herbalist, hifi, houseware, hunting, insurance,
+    interior_decoration, jewelry, laundry, leather, locksmith, kiosk, kitchen, lamps, lottery, mall,
+    massage, medical_supply, mobile_phone, model, motorcycle, music, musical_instrument,
+    nutrition_supplements, newsagent, optician, organic, outdoor, paint, pastry, perfumery, photo,
+    pyrotechnics, rediotechnics, seafood, second_hand,secruity, shoes, spices, sports, stationery,
+    supermarket, swimming_pool, tailor, tattoo, tea, ticket, tiles, tobacco, toys, travel_agency, trophy,
+    tyres, variety_store, video, video_games, watches, weapons, wine, pet))
+    or (tourism in (alpine_hut, apartment, aquarium, artwork, attraction, camp_site, caravan_site,
+    chalet, gallery, museum, guest_house, hostel, hotel, motel, picnic_site, theme_park, viewpoint,
+    wilderness_hut, zoo))
+    or (vaccination = covid19)
+
+landmarks:
+  name: POIs for landmarks
   description: |
     A lot of different objects such related to natural features
-    transportation and amenities in a city.
+    transportation and amenities in a city. Definition taken from
+    Waterproofing data project.
```

Comment:
Whats the waterproofing project? (From an outsider perspective/ If you are not working at HeiGit).
If possible please provide a link to the source of the definition.
This could also be a new key: E.g. `source: url.de`

Reply:
https://www.geog.uni-heidelberg.de/gis/sketchmaptool.html